tests/test_cli.py

```diff
@@ -2165,6 +2170,31 @@ def test_retain_newline(tmpdir, configfile, newline):
     # and that it is of the right type
     assert new_config.endswith(b"[bumpversion:file:file.py]" + newline)
 
+def test_no_configured_files(tmpdir, vcs):
```

Comment:
Please keep 2 empty lines between functions:
```suggestion

def test_no_configured_files(tmpdir, vcs):
```